### PR TITLE
SEO-173276-WPF-Image-Link-Changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
Please find changes made under image section.
Regards,
Scholar.

### CODE REVIEW

File: wpf/Sunburst-Chart/Getting-Started.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ This section explains you the steps required to populate the sunburst chart with
...
 
 Drag and drop the SfSunburstChart control from the Toolbox to your application,
 
-![Visual Studio Toolbox](Getting-Started_images/gettingstarted2.png)
+![WPF Sunburst Chart Visual Studio Toolbox](getting-started_images/wpf-sunburst-chart-toolbox.png)
 
 Now the Syncfusion.SfSunburstChart.WPF reference is added to the application references and the namespace code is generated in MainWindow.xaml as below.
 
-![Project Solution Window contains SfSunburstChart reference](Getting-Started_images/gettingstarted3.png)
+![WPF Sunburst Chart Project Solution reference](getting-started_images/wpf-sunburst-chart-project-solution.png)
```
Comment (MallikaRK) [inline wpf/Sunburst-Chart/Getting-Started.md:31]: Image alt text name casing is not proper. Should add Caps letter in starting of the text, control name, platform name, other text should be in small casing. So, change it.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/RichTextBox/Getting-Started.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ N> Starting with v16.2.0.41 (2018 Vol 2), if you reference Syncfusion assemblies
 After adding the aforementioned assembly references to your application, you can add SfRichTextBoxAdv as any normal control.
 You can either drag the SfRichTextBoxAdv control from the toolbox window to the Design view or directly define the control in XAML view.
 
-![WPF RichTextBox Control](Getting-Started_images/wpf-richtextbox-control.jpeg)
+![WPF RichTextBox Application](getting-started_images/wpf-richtextbox-application.jpeg)
```
[the remark this answers is not in the document]
Comment (Scholar-jaruha) [reply wpf/RichTextBox/Getting-Started.md:40]: I have reverted the changes

File: wpf/Sunburst-Chart/Getting-Started.md (hunk 1 above)
Comment (Scholar-jaruha) [reply wpf/Sunburst-Chart/Getting-Started.md:31]: changed

File: wpf/Scheduler/Overview.md [hunk 3]
```diff
@@ -10,26 +10,26 @@ documentation: ug
 
 The [WPF Scheduler](https://www.syncfusion.com/wpf-controls/scheduler) control is used to schedule and manage appointments through an intuitive user interface, similar to the Outlook calendar.
 
-![WPF scheduler FirstDayOfWeek](GettingStarted_images/wpf-scheduler-firstdayofweek.png)
+![WPF scheduler rirstdayofweek](overview_images/wpf-scheduler-firstdayofweek.png)
```
Comment (MallikaRK) [inline wpf/Scheduler/Overview.md:13]: Spelling mistake in alt text.
control name should be casing.
image name should be proper, give hyphen.

File: wpf/Tab-Splitter/Overview.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 TabSplitter is similar to VS 2008 style Split view of Tabbed Groups.
 
-![Overview of TabSplitter](Overview_images/overview.png)
+![Overview in WPF TabSplitter](overview_images/wpf-tab-splitter-overview.png)
```
Comment (MallikaRK) [inline wpf/Tab-Splitter/Overview.md:14]: overview of is suitable. Revert your changes in Alt text.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Scheduler/Overview.md (hunk 3 above)
Comment (Scholar-jaruha) [reply wpf/Scheduler/Overview.md:13]: spelling error corrected

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**